SFR/Assembly/outer_assembly/mesh.i

```diff
@@ -0,0 +1,67 @@
+!include ../../common_input.i
+
+[Mesh]
+    [Pin]
+      type = PolygonConcentricCircleMeshGenerator
+      num_sides = 6
+      num_sectors_per_side = '${NUM_SECTORS} ${NUM_SECTORS} ${NUM_SECTORS} ${NUM_SECTORS} ${NUM_SECTORS} ${NUM_SECTORS}'
+      ring_radii = '${r_fuel} ${fparse r_fuel + t_gap} ${fparse r_fuel + t_gap + t_clad}'
+      ring_intervals = '${FUEL_RADIAL_DIVISIONS} 1 1'
+      polygon_size = ${fparse pitch / 2.0}
+  
+      ring_block_ids = '0 1 2 3'
+      ring_block_names = 'fuel_center fuel gap cladding'
+      
+  
+      flat_side_up = false
+      quad_center_elements = false
+      preserve_volumes = true
+  
+      create_outward_interface_boundaries = true
+    []
+    [Assembly_2D]
+        type = PatternedHexMeshGenerator
+        inputs = 'Pin'
+        pattern ='        0 0 0 0 0 0 0 0 0;
+                         0 0 0 0 0 0 0 0 0 0;
+                        0 0 0 0 0 0 0 0 0 0 0;
+                       0 0 0 0 0 0 0 0 0 0 0 0;
+                      0 0 0 0 0 0 0 0 0 0 0 0 0;
+                     0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                    0 0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                   0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                   0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                    0 0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                     0 0 0 0 0 0 0 0 0 0 0 0 0 0;
+                      0 0 0 0 0 0 0 0 0 0 0 0 0;
+                       0 0 0 0 0 0 0 0 0 0 0 0;
+                        0 0 0 0 0 0 0 0 0 0 0;
+                         0 0 0 0 0 0 0 0 0 0;
+                          0 0 0 0 0 0 0 0 0;'
+
+         hexagon_size = '${fparse edge_length}'
```

Comment:
You might need to specify `hexagon_size_style` here. IIRC the [default](https://mooseframework.inl.gov/source/meshgenerators/PatternedHexMeshGenerator.html) is `apothem` (in which case you'd give one half the flat to flat distance), but I think you'd want to use `radius` if you're supplying `edge_length`